examples/sensors/main.c

```diff
@@ -37,31 +41,52 @@ static void timer_fired(__attribute__ ((unused)) int arg0,
   int ninedof_accel_x = 0, ninedof_accel_y = 0, ninedof_accel_z = 0;
   int ninedof_magneto_x = 0, ninedof_magneto_y = 0, ninedof_magneto_z = 0;
   int ninedof_gyro_x = 0, ninedof_gyro_y = 0, ninedof_gyro_z = 0;
+  unsigned proxReading = 0;
```

Comment:
Minor consistency nit: The style in libtock_c is `_` rather than camel case

Reply:
Hello professor @ppannuto.  Sorry for the delay.  My PR for the kernel proximity interface was just approved which I was waiting for before I addressed this.  Ive commented the interface a little more and changed it (proximity is now returned as a `uint8_t` and have addressed all of your comments.  Please let me know if I accidentally missed anything.